### ISSUE TITLE
split memory into memory + memory_unit

### ISSUE THREAD

**Issue by jahilton** [HumanCellAtlas/metadata-schema#1169]
**For which schema is a change/update being suggested?**

analysis_process.json

**What should the change/update be?**

change `memory` field to capture _just_ the value of the memory
add `memory_unit` field to capture the units of memory
requires addition of a new *_unit_ontology.json

**Why is the change requested?**

consistency with other metadata capturing value+unit (height, weight, age, etc)

**Comment by mshadbolt** [comment]
Is the analysis/pipelines team aware of this change and prepared to fill in these fields in the metadata after running their pipelines?

**Comment by jahilton** [comment]
I don't know how much heads-up pipelines will need to make the necessary requirements.

I think there needs to be a huddle with pipelines + metadata because I imagine there should be more updates in terms of consistency & thoroughness, so probably don't make this change until the full scope of changes is realized
(and Matrix Service should be included in the huddle for the time in the future when their metadata also need to be captured)

**Comment by lauraclarke** [comment]
While I understand the consistency goal here. It would be useful to understand how useful a field this is to consumers of the analysis metadata vs the cost of pipelines changing their code and migrating all existing analysis process depositions to use the new schema

**Comment by zperova** [comment]
@jahilton is there any update on this from the pipelines team?

**Comment by jahilton** [comment]
Nope. I would anticipate this being saved on our 'wish list' and incorporated only once there's a full analysis metadata review, which may not happen until analysis AUDRs are possible (so we can migrate existing metadata into the new schema).

**Comment by zperova** [comment]
thanks @jahilton
